IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeCatenaryStay/DocPropertySet.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypeCatenaryStay" UniqueId="b15eff21-bd3f-df08-8b2a-e9722e73b14d" ApplicableType="IfcMember/BRACE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AssemblyInstruction_3W5i6nbkP9NBt_M6PTB5jW" />
+		<DocProperty xsi:nil="true" href="InstallationAngle_22EVJmQpX1LByB190D6_1E" />
+		<DocProperty xsi:nil="true" href="InstallationHeight_2fIpSt1Hb4dOgwSFym6qxV" />
+		<DocProperty xsi:nil="true" href="StayLength_0q7QQ5ddHF39tvrVe83yvP" />
```

Comment:
this is a quantity so should be stored as such in a Qto (there already is a length), in addition MemberCommon already has Span, Slope and Roll properties that can cover a few of the above.